app/graphql/types/comment_type.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class CommentType < Types::BaseObject
+    field :id, ID, null: false
+    field :user_id, Integer
+    field :task_id, Integer
```

Comment:
Remove it. CommentType should return UserType and TaskType, not ids of another models